recipes/auto_rxn/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "auto_rxn" %}
+{% set version = "2023.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/auto_rxn-{{ version }}.tar.gz
+  sha256: 616825da1be19de17f01065619d7af57481083e691ec42a7fef4931b96b53317 
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - "auto_rxn=auto_rxn.__main__:main"
+
+requirements:
+  host:
+    - python >=3.7
+    - flit-core
+    - pip
+  run:
+    - python >=3.7
+    - appdirs
+    - click
+    - happi
+    - pandas
+    - bluesky
+    - suitcase-csv
+    - packaging
+
+test:
+  imports:
+    - auto_rxn
+  commands:
+    #- pip check
```

Comment:
Can you add a comment in the recipe to why `pip check` is commented out.

Reply:
you bet @ocefpaf